dev/integration_tests/flutter_gallery/README.md

```diff
@@ -0,0 +1,11 @@
+# Flutter gallery
+
+An older copy of the Flutter gallery demo application used for integration testing.
+
+For the current Flutter gallery app sample, see the samples repo:
+https://github.com/flutter/samples/tree/master/gallery
```

Comment:
The gallery just moved out of DevRel's repo and into https://github.com/flutter/gallery

There's also a hosted version at flutter.github.io/gallery.

Reply:
Links updated